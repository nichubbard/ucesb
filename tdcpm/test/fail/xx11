/* ### Unit mismatch */

/* Fail due to wrong unit. */
s[0].k = 1 ns;

