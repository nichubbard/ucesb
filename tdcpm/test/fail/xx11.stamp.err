0x5583b07fb340 1.0000
Unit mismatch, cannot assign.
