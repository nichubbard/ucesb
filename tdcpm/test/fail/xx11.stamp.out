s(1).k = 1 ns;
